packages/wallet/frontend/next.config.js

```diff
@@ -2,6 +2,15 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true'
 })
 
+let NEXT_PUBLIC_FEATURES_ENABLED = 'true'
+
+if (
+  process.env.NODE_ENV === 'production' &&
+  process.env.GATEHUB_ENV === 'staging'
```

Comment:
staging or sandbox?

Reply:
Oh, it's sandbox in backend. Will update.